lib/ok.ex

```diff
@@ -122,33 +122,93 @@ defmodule OK do
 
   ## Examples
 
-      iex> OK.is_success?({:ok, "some value"})
+      iex> OK.success?({:ok, "some value"})
       true
 
-      iex> OK.is_success?({:error, :some_reason})
+      iex> OK.success?({:error, :some_reason})
       false
   """
-  @spec is_success?({:ok, a}) :: true when a: any
-  @spec is_success?({:error, reason}) :: false when reason: any
-  def is_success?({:ok, _value}), do: true
-  def is_success?({:error, _reason}), do: false
+  @spec success?({:ok, a}) :: true when a: any
+  @spec success?({:error, reason}) :: false when reason: any
+  def success?({:ok, _value}), do: true
```

Comment:
This might be the right thing to do but it is now a breaking change, not sure if it's worth a 3.0 release to fix